Wikipedia/Networking/Fetchers/SavedArticlesFetcher.h

```diff
@@ -17,14 +23,14 @@
 + (SavedArticlesFetcher*)sharedInstance;
```

Comment:
It we are creating an ivar to hold the reference to the long living saved fetcher (currently in the app delegate) - we should remove the shared access to it, since it is now duplicative.